lib/event-service.js

```diff
@@ -0,0 +1,59 @@
+var EVENTSERVICE_SERVICE_UUID          = 'E95D93AF251D470AA062FA1922DFA9A8'.toLowerCase();
+var MICROBITEVENT_CHARACTERISTIC_UUID = 'E95D9775251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var CLIENTREQUIREMENTS_CHARACTERISTIC_UUID = 'E95D23C4251D470AA062FA1922DFA9A8'.toLowerCase();
+var CLIENTEVENT_CHARACTERISTIC_UUID = 'E95D5404251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var microbit_event_subscribed = 0;

```

Comment:
Apologies for not stating the reason for this suggestion ...

Basically, if more than one micro:bit is used in a Node.js app, due the way scoping works, the variable will be shared between both instances - making it a member var would make it independent per instance :)